### PR TITLE
Hesa list supported licenses

### PR DESCRIPTION
**Pull request by hesa**
With these commits (I hope) we're adding support for listing all supported licenses

### CODE REVIEW

File: tests/test_customdb.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
```
Comment (priv-kweihmann) [inline tests/test_customdb.py:1]: Not really needed IMO

Comment (hesa) [reply tests/test_customdb.py:1]: It is copied from the other test files. You want me to remove it from all files?

However, I fixed a typo in the name of the class `TestCustomMatrix`.

Comment (priv-kweihmann) [reply tests/test_customdb.py:1]: Yeah pls remove it from all the files

Comment (hesa) [reply tests/test_customdb.py]: done

**Review by priv-kweihmann — APPROVED** [review]

**Pull request merged**